prow/github/types.go

```diff
@@ -80,6 +80,7 @@ type PullRequest struct {
 	RequestedReviewers []User            `json:"requested_reviewers"`
 	Assignees          []User            `json:"assignees"`
 	Merged             bool              `json:"merged"`
+	MergeSHA           string            `json:"merge_commit_sha"`
```

Comment:
This should probably be type `*string`, and check if it's `null` before using. Also, please document it, since it's not trivial.